test/consumer.test.ts

```diff
@@ -484,6 +484,21 @@ describe('Consumer', () => {
       sqs.send.calledOnceWith(mockReceiveMessage);
     });
 
+    it("doesn't consume more messages when called multiple times after stopped", () => {
```

Comment:
This passes with the latest version of consumer, unsure if it's a valid test or not, need to think on it.